src/python/pants/engine/platform.py

```diff
@@ -15,6 +15,7 @@ class Platform(Enum):
     linux_x86_64 = "linux_x86_64"
     macos_arm64 = "macos_arm64"
     macos_x86_64 = "macos_x86_64"
+    linux_arm64 = "linux_arm64"
```

Comment:
You'll need to update all uses of `Platform` to include `linux_arm64` - this will tell Pants how to install native binaries like Protoc. See https://www.pantsbuild.org/v2.7/docs/reference-protoc#section-known-versions for instructions on computing the SHA256 and # bytes.

We won't be testing Pants aarch64 in CI because GitHub Actions doesn't have it...so that means it's going to be up to contributors to ensure things work. When adding a new binary, it would be really helpful if you're willing to test that things work locally. Lmk if I can help with figuring out how to test each thing (usually running `./pants test` on its related files should do the trick).